### ISSUE TITLE
LoginPage - remove aria-label from checkbox

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1737]
The checkbox in the LoginPage component includes an aria-label in addition to a traditional `<label>`. This attribute should be removed, since we should be using the string defined inside `<label>` only.
